pkg/cluster/initialize.go

```diff
@@ -58,6 +61,31 @@ func (u *simpleUpgrader) Initialize(ctx context.Context, cs *api.OpenShiftManage
 	return b.CreateBlockBlobFromReader(bytes.NewReader(csj), nil)
 }
 
+func (u *simpleUpgrader) CreateStorageAccount(ctx context.Context, cs *api.OpenShiftManagedCluster, accountName string) error {
+	parameters := azstorage.AccountCreateParameters{
+		Sku: &azstorage.Sku{
+			Name: azstorage.StandardLRS,
+		},
+		Kind:     azstorage.Storage,
+		Location: &cs.Location,
+	}
+	if accountName == cs.Config.ConfigStorageAccount {
+		parameters.Tags = map[string]*string{
+			"type": to.StringPtr("config"),
+		}
+	}
+	future, err := u.accountsClient.Create(ctx, cs.Properties.AzProfile.ResourceGroup, accountName, parameters)
```

Comment:
we've moved WaitForCompletionRef's into the API layer to ease unit testing.  Can you do the same with this one?

Reply:
done